src/pypi_attestation_models/_impl.py

```diff
@@ -72,6 +76,15 @@ class Attestation(BaseModel):
     is the raw bytes of the signing operation over the attestation payload.
     """
 
+    def verify(self, verifier: Verifier, policy: VerificationPolicy, dist: Path) -> None:
+        """Verify against an existing Python artifact.
+
+        On failure, raises `sigstore.errors.InvalidAttestationError`.
```

Comment:
Is this correct? I think `InvalidAttestationError` isn't a type in `sigstore-python`

Reply:
True, the `docstring` is wrong here. Do we want to wrap sigstore's `VerificationError` with our own exception class, or just use sigstore's?

Reply:
Hmm, I suppose we should have it be our own -- wrapping makes sense to me 🙂

Reply:
fixed!